### PR TITLE
[new release] cairo2, cairo2-pango and cairo2-gtk (0.6.3)

### PR DESCRIPTION
**Pull request by Chris00**
Binding to Cairo, a 2D Vector Graphics Library

- Project page: <a href="https://github.com/Chris00/ocaml-cairo">https://github.com/Chris00/ocaml-cairo</a>
- Documentation: <a href="https://Chris00.github.io/ocaml-cairo/doc">https://Chris00.github.io/ocaml-cairo/doc</a>

##### CHANGES:

- Fix bug in `Path.fold`.
- Extend `Image.get_data*` to bigarrays with externally managed payload.

### CODE REVIEW

File: packages/cairo2-pango/cairo2-pango.0.6.3/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+           "Pierre Hauweele <pierre@hauweele.net>" ]
+license: "LGPL-3.0"
+homepage: "https://github.com/Chris00/ocaml-cairo"
+dev-repo: "git+https://github.com/Chris00/ocaml-cairo.git"
+bug-reports: "https://github.com/Chris00/ocaml-cairo/issues"
+doc: "https://Chris00.github.io/ocaml-cairo/doc"
+tags: ["Cairo" "stroke" "drawing" "tutorial"]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/cairo2-pango/cairo2-pango.0.6.3/opam:12]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/cairo2-gtk/cairo2-gtk.0.6.3/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+           "Pierre Hauweele <pierre@hauweele.net>" ]
+license: "LGPL-3.0"
+homepage: "https://github.com/Chris00/ocaml-cairo"
+dev-repo: "git+https://github.com/Chris00/ocaml-cairo.git"
+bug-reports: "https://github.com/Chris00/ocaml-cairo/issues"
+doc: "https://Chris00.github.io/ocaml-cairo/doc"
+tags: ["Cairo" "stroke" "drawing" "tutorial"]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/cairo2-gtk/cairo2-gtk.0.6.3/opam:12]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/cairo2/cairo2.0.6.3/opam [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+           "Pierre Hauweele <pierre@hauweele.net>" ]
+license: "LGPL-3.0"
+homepage: "https://github.com/Chris00/ocaml-cairo"
+dev-repo: "git+https://github.com/Chris00/ocaml-cairo.git"
+bug-reports: "https://github.com/Chris00/ocaml-cairo/issues"
+doc: "https://Chris00.github.io/ocaml-cairo/doc"
+tags: ["Cairo" "stroke" "drawing" "tutorial"]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/cairo2/cairo2.0.6.3/opam:12]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Thanks

**Comment by Chris00** [comment]
Thanks for the comments — I keep forgetting this.

**Pull request merged**